Xamarin.Forms.Platform.UAP/CollectionView/ItemsViewRenderer.cs

```diff
@@ -97,6 +97,12 @@ protected virtual void UpdateItemsSource()
 
 			var itemsSource = Element.ItemsSource;
 
+			if (itemsSource == null)
+			{
+				_collectionViewSource = null;
+				return;
```

Comment:
This return avoid to do the item inflate and binding process, which is correct. But, it will also prevent the ItemsSource from being set here https://github.com/xamarin/Xamarin.Forms/blob/c1eea05b9976b0db00bf155f3754ba1aac072fa6/Xamarin.Forms.Platform.UAP/CollectionView/ItemsViewRenderer.cs
I think it is correct, because EmptyView is shown with Null or empty ItemsSource, but I have doubts, It's like that, right?.